queries/data-transformations/data_transformations/crossref_transform.py

```diff
@@ -0,0 +1,261 @@
+import argparse
+import logging
+import os
+import pathlib
+
+import polars as pl
+from polars._typing import SchemaDefinition
+
+from cli import handle_errors, add_common_args, validate_common_args
+from pipeline import process_files_parallel
+from transformations import remove_markup, normalise_identifier, date_parts_to_date
+from utils import read_jsonls, validate_directory, extract_gzip
+
+logger = logging.getLogger(__name__)
```

Comment:
I will leave this up to you to decide when you want to do it, but I'd like for the logs to use the Elastic Common Schema. We will eventually pipe logs into OpenSearch (once Ashley has time to get that setup for us) and the ESC format is ideal for that. 

It looks like the `ecs-logging-python` package can be used for that.

For example here's what one from the Apollo server looks like:
`{"level":20,"time":1746046022412,"pid":25,"hostname":"3becdb5ae747","requestId":"f8946fb17f55a94b7ca28bdd52f79dd3","jti":"15-1746045222223","userId":15,"msg":"updateProject resolver, sql: SELECT * FROM plans WHERE projectId = ?, vals: 11"}`

Reply:
Yeah sure I can do that, it would be good to do it in a separate ticket.